### ISSUE TITLE
Training results doesn't match the result mentioned in the paper

### ISSUE THREAD

**Issue by YasmineHamdy** [arpit-jadon/FireNet-LightWeight-Network-for-Fire-Detection#7]
Hi,
First of all thank you for sharing the code,
i trained the model using the train script and, 
When i compare the output results of testing the new trained model , it doesn't match the result mentioned in the paper.

Could you please explain why?

**Comment by akshay-varshney** [comment]
Hi @YasmineHamdy,
Thanks for checking out our work. We would like to know more information about your results.
>Did you used the same training data set which we have mentioned?
>Also how many epochs did you run the mode training? 
> What was your training accuracy? 

Also we have added our trained model, you can also use the same. (FireNet-LightWeight-Network-for-Fire-Detection/Trained_Model_FIreNet/)

Thanks.
